variant_annotator.py

```diff
@@ -0,0 +1,90 @@
+            annotation_json_data = response.json()[0]  # json object is returned in a one item list. just return the zero item for ease of reading
+
+            # Put data in a dict for later processing. Dict is good as we can reference by name later.
+            # TODO: check for missing data? Not sure.
+            annotation_dict = {
+                'variant': variant_in,
+                'assembly_name': annotation_json_data['assembly_name'],
+                'seq_region_name': annotation_json_data['seq_region_name'],
+                'start': annotation_json_data['start'],
+                'end': annotation_json_data['end'],
+                'most_severe_consequence': annotation_json_data['most_severe_consequence'],
+                'strand': annotation_json_data['strand'],
+                'gene_symbol': annotation_json_data['transcript_consequences'][0]['gene_symbol']
+            }
+            # print(variant_in, file=sys.stdout)
+            return annotation_dict
+        else:
+            print('The input value: "' + variant_in + '" failed with a response of: "' + str(response.status_code) + '"', file=sys.stderr)
+            return {} # This could also return None but we returns a dict for funtion output consistancy
+            
+    def create_output_file(self)-> None:
+        # TODO: fix the path that this file gets written to
+        with open(self.filename_out, 'w', newline='') as tsvfile:
+            writer = csv.writer(tsvfile, delimiter='\t', lineterminator='\n')
+            writer.writerow(['variant', 'assembly_name', 'seq_region_name', 'start', 'end', 'most_severe_consequence', 'strand', 'genes'])
+            
+
+    def process_results_to_file(self, annotation_in) -> None:
+        # TODO: fix the path that this file gets written to
+        with open(self.filename_out, 'a', newline='') as tsvfile_data:
+            writer = csv.writer(tsvfile_data, delimiter='\t', lineterminator='\n')
+            # writerow takes single arg, so putting values in a single array element, I could probably just pass annotation_in, but the order of elements is not guaranteed to match the header.
+            writer.writerow(
+                [
+                    annotation_in['variant'],
+                    annotation_in['assembly_name'],
+                    annotation_in['seq_region_name'],
+                    annotation_in['start'],
+                    annotation_in['end'],
+                    annotation_in['most_severe_consequence'],
+                    annotation_in['strand'],
+                    annotation_in['gene_symbol']
+                ]
+            )
+    
+    def print_output_file_sdout(self):
+        with open(self.filename_out, 'r') as f:
+            print(f.read(), file=sys.stdout)
+
+####################################################
+# Instantiate the Object  and Process the Variants #
+####################################################
+
+v = VariantAnnotator()
```

Comment:
Would like to see this with at least `if __name__ == "__main__":`  style logic, so that these functions could be imported separately. As is, this limits this to only being a script, rather than both a script and a set of functions to be used elsewhere. 

Even better would be to make this a function, `cli`, and link that to the package via `console.scripts` in the pyproject.toml, see https://setuptools.pypa.io/en/latest/userguide/entry_point.html